### ISSUE TITLE
About distributed training

### ISSUE THREAD

**Issue by wjn922** [IBM/action-recognition-pytorch#2]
Hi, is this project able to use distributed training in multiple nodes?

**Comment by chunfuchen** [comment]
Yes, you can refer to [here](https://github.com/pytorch/examples/tree/master/imagenet) to see how to start distributed training
